lib/domain.mjs

```diff
@@ -0,0 +1,7 @@
+import domain from 'domain';
```

Comment:
Should we export deprecated modules at all?

Reply:
+0 to removing them, not blocking on it